Assets/MRTK/SDK/Experimental/PulseShader/Scripts/SurfacePulseOnSelect.cs

```diff
@@ -0,0 +1,204 @@
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Utilities;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.SurfacePulse
+{
+	[AddComponentMenu("Scripts/MRTK/SDK/SurfacePulse")]
+	public class SurfacePulseOnSelect : MonoBehaviour, IMixedRealityPointerHandler
+	{
+		[Tooltip("Shader parameter name to drive the pulse radius")]
+		public string ParamName = "_Pulse_";
+
+		[Tooltip("Shader parameter name to set the pulse origin, in local space")]
+		public string OriginParamName = "_Pulse_Origin_";
+
+		[Tooltip("How long in seconds the pulse should animate")]
+		public float PulseDuration = 5f;
+
+		[Tooltip("How long to wait in seconds between pulses, when pulsing is active")]
+		public float PulseRepeatDelay = 5f;
+
+		[Tooltip("Minimum time to wait between each pulse")]
+		public float PulseRepeatMinDelay = 1f;
+
+		[Tooltip("Automatically begin repeated pulsing")]
+		public bool bAutoStart = false;
```

Comment:
Please update all the fields to be `private` with `public` property accessors if needed, to follow [the repo's guidelines on encapsulation](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#encapsulation).

There are also a couple that appear to use Hungarian notation (like this bool starting with `b`), which isn't a standard we've used in this repo. They should be updated to just the names.

Reply:
Update applied to both SurfacePulseOnSelect.cs and SurfacePulse.cs script!